Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -660,3 +639,35 @@ internal extension CFURLSessionEasyCode {
         throw self
     }
 }
+
+#if os(Android)
+extension URLSession {
+    fileprivate static var _sslCertificateAuthorityFilePathPtr: UnsafeMutablePointer<Int8>?
+    /// You can bypass SSL peer verification altogether (not advised)
+    public static var verifyPeerSSLCertificate = true
+    /// See https://curl.haxx.se/docs/sslcerts.html
+    /// For SSL to work you need a "cacert.pem" to be accessible at the path
+    /// provided by setting URLSession.sslCertificateAuthorityFile to a URL.
+    /// Downloadable here: https://curl.haxx.se/docs/caextract.html
+    public static var sslCertificateAuthorityFile: URL {
+        set(value) {
+            verifyPeerSSLCertificate = true
```

Comment:
While I enjoy our online jousts this one is getting a bit OTT :) An app dev has two choices when they develop an app: Set verifyPeerSSLCertificate to false (not advised but can be useful if you self-sign during development) or set sslCertificateAuthorityFile to a valid URL of a pem fille. If they do both the more secure option prevails. I’d realy like to keep this simple rather than create a minefiled of interlocks that need to be implemented and documented.